pkg/cmd/check_banned.go

```diff
@@ -0,0 +1,115 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/banneduser"
+	"github.com/codeready-toolchain/toolchain-common/pkg/hash"
+	"github.com/kubesaw/ksctl/pkg/client"
+	clicontext "github.com/kubesaw/ksctl/pkg/context"
+	"github.com/kubesaw/ksctl/pkg/ioutils"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/errors"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NewCheckBannedCommand() *cobra.Command {
+	var mur string
+	var signup string
+	var email string
+
+	command := &cobra.Command{
+		Use:   "check-banned [--mur|-m <mur-name>] | [--signup|-s <signup-name>] | [--email|-s <email>]",
+		Short: "Unban the user so that they can use the platform again",
+		Long:  "Unban the user that previously registered with the provided email so that they can start using the platform again.",
+		Args:  cobra.ExactArgs(1),
```

Comment:
```suggestion
```
I think we can remove this so your better error message from the `validateCheckBannedInput()` func shows instead.